pyxlsb/cell.py

```diff
@@ -17,15 +17,23 @@ def v(self):
     def f(self):
         return self.formula
 
+    @property
+    def s(self):
+        return self.style
+
 class Cell(DeprecatedCellMixin):
-    __slots__ = ('row', 'col', 'value', 'formula')
+    __slots__ = ('row_cls', 'row', 'col', 'value', 'value_conv', 'formula', 'style_num', 'style_fmt')
 
-    def __init__(self, row, col, value=None, formula=None):
-        self.row = row
+    def __init__(self, row, col, value=None, value_conv=None, formula=None, style_num=None, style_fmt=None):
+        self.row_cls = row
+        self.row = row.num
         self.col = col
         self.value = value
+        self.value_conv = value_conv
```

Comment:
We should expose this as a property or a getter, potentially caching the value locally, but in any case we should produce it lazily.